### DISCUSSION TITLE
Quero participar dos testes do App

### DISCUSSION THREAD

**Discussion by orodrigogo** [diego3g/rsxp-2023#111] [General]
Fala galera! 👋
Aqui vamos centralizar as definições e documentação dos testes do App. 

Já estamos caminhando para conclusão da primeira versão do App. Desta forma, em breve, teremos um conjunto de funcionalidades disponíveis para testar. O teste é uma etapa fundamental para identificar os problemas e trabalharmos na correção. 🤗 

Para adiantar, podemos iniciar a etapa de colher as pessoas interessadas em participar dos testes do App. Você pode inscrever-se para testar a versão Android, iOS ou ambas. 

Assim que tivermos a primeira versão disponível. Na próxima etapa, será liberada a versão do App para os testadores (a) conforme a seguir. 

### Quero testar a versão Android
No Android vamos começar com testes interno que permite liberar imediatamente o App para testar. 

Se você deseja torna-se um testador (a) do App, inscreve-se neste link informando a sua conta Google (@gmail.com). 
Neste tipo de teste o limite é 100 testadores. Sendo assim, quando atingir esse número eu sinalizado aqui para vocês. 

### Quero testar a versão iOS
No iOS vamos utilizar o **TestFlight** para testar o App. Se você pretende colaborar testando a versão iOS do App instale em seu dispositivo o TestFlight clicando nesse link:

[https://apps.apple.com/us/app/testflight/id899247664](https://apps.apple.com/us/app/testflight/id899247664)
Neste tipo de teste o limite é 10.000 testadores.

### Quero participar
Para facilitar, criei um Google Forms para coletar os interessados em testar o App. Para participar, inscreva-se neste link: https://forms.gle/r7AMELygV84gruJF7

Fique ligado (a) no e-mail que você utilizou para realizar os testes. Pois, enviaremos as instruções para os próximas etapas por lá também.

E claro, aqui é o lugar para contribuir com suas sugestões! 💜

**Comment by gabrielbriks** [comment]
@rodrigorgtic  o Google Form está solicitando Permissão para acessa-lo

**Reply by orodrigogo** [reply]
Agora sim, liberado. 😄
